articles/quickstart/spa/angular2/06-token-renewal.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Silent Token Renewal
+description: This tutorial demonstrates how to add automatic access_token renewal to an Angular 2+ application with Auth0
+budicon: 448
+---
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-angular-samples',
+  path: '06-Token-Renewal',
+  requirements: [
+    'Angular 2+'
+  ]
+}) %>
+
```

Comment:
Add a short paragraph for preamble. Something like:

"As a security measure, it is recommended that the lifetime of a user's `access_token` be kept short. When you create an API in the Auth0 dashboard, the default lifetime is 7200 seconds (2 hours).

This short lifetime is good for security, but it isn't great for user experience. You will likely want to provide a way for your users to automatically get a new `access_token` so that their client-side session can be kept alive. This can be done with **Silent Authentication**.